### DISCUSSION TITLE
Hello, there were a few small questions, I am a beginner

### DISCUSSION THREAD

**Discussion by testinghere57564** [qdm12/gluetun#785] [Q&A]
Can I bring proxy to external ip? Can I run several of these connections at the same time?
Thanks for the answer.

**Comment by qdm12** [comment]
> Can I bring proxy to external ip?

Not sure what you mean, but yes you can connect to the Shadowsocks or http proxy through your public IP address. Although http proxy is not encrypted so not recommended from outside your local network.

> Can I run several of these connections at the same time?

To my biggest surprise of like 2018, YES !
